src/en/nightscans/src/eu/kanade/tachiyomi/extension/en/nightscans/NightScans.kt

```diff
@@ -5,7 +5,7 @@ import eu.kanade.tachiyomi.network.interceptor.rateLimit
 import okhttp3.OkHttpClient
 import java.util.concurrent.TimeUnit
 
-class NightScans : MangaThemesia("NIGHT SCANS", "https://nightscans.net", "en", "/series") {
+class NightScans : MangaThemesia("NIGHT SCANS", "https://night-scans.com", "en", "/series") {
```

Comment:
inherit from `MangaThemesiaAlt` as it contains support for dynamic urls (numeric part in manga url)
![image](https://github.com/keiyoushi/extensions-source/assets/48650614/49425033-1330-490f-9099-c7ab652f55c1)